src/adhocracy_aula/setup.cfg

```diff
@@ -0,0 +1,20 @@
+[compile_catalog]
+directory = adhocracy_aula/locale
+domain = adhocracy_aula
+statistics = true
+
+[extract_messages]
+add_comments = TRANSLATORS:
+output_file = adhocracy_aula/locale/adhocracy_aula.pot
+width = 80
+
+[init_catalog]
+domain = adhocracy_aula
+input_file = adhocracy_aula/locale/adhocracy_aula.pot
+output_dir = adhocracy_aula/locale
+
+[update_catalog]
```

Comment:
We use the script bin/i18n to manage translation file and  have no setuptool integration based on babel:
https://github.com/liqd/adhocracy3/blob/master/docs/development.rst#update-translations-backend

So setup.cfg ist not used at all and we can remove it from the scaffold files, or did I miss something?